{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,136 +1,142 @@
 FROM python:3.6-alpine as base
-
-LABEL maintainer={{ cookiecutter.github_username }}
-
+#
 #  USAGE:
 #     cd sercices/{{ cookiecutter.project_slug }}
 #     docker build -f Dockerfile -t {{ cookiecutter.project_slug }}:prod --target production ../../
 #     docker run {{ cookiecutter.project_slug }}:prod
 #
 #  REQUIRED: context expected at ``osparc-simcore/`` folder because we need access to osparc-simcore/packages
 
-# non-root user 'scu'
-RUN adduser -D -u 8004 scu
+LABEL maintainer={{ cookiecutter.github_username }}
+
+# simcore-user uid=8004(scu) gid=8004(scu) groups=8004(scu)
+RUN adduser -D -u 8004 -s /bin/sh -h /home/scu scu
 
 RUN apk add --no-cache \
       su-exec
 
-ENV HOME /home/scu
-# TODO: Workaround for https://github.com/pypa/pip/issues/6197:
-#   - Added explicit --cache-dir that is removed at the end of the build
-#   - NOTE: After every PIP call, add rm -rf /tmp/.pipcache
-ENV PIP /home/scu/.venv/bin/pip3 --cache-dir=/tmp/.pipcache
+ENV PATH "/home/scu/.local/bin:$PATH"
+
+# All SC_ variables are customized
+ENV SC_PIP pip3 --no-cache-dir
+ENV SC_BUILD_TARGET base
 
 EXPOSE 8080
 
-HEALTHCHECK --interval=30s \
-            --timeout=120s \
-            --start-period=30s \
-            --retries=3 \
-            CMD python3 ${HOME}/docker/healthcheck_curl_host.py "http://localhost:8080/{{ cookiecutter.openapi_specs_version }}/"
 
 # -------------------------- Build stage -------------------
-# Common stage between production and development targets
-#
-# + /home/scu/              $HOME
-#    + packages
-#        ...
-#    + services/{{ cookiecutter.project_slug }}
-#       + src
-#       + tests
+# Installs build/package management tools and third party dependencies
 #
+# + /build             WORKDIR
 #
-# TODO: straight copying python packages bring unnecessary files (e.g. __pycache__) -> dockerignore!
-#       could copy and then python setup.py install OR git clone into the container.
-#       This applies for both
-#
-#  SEE https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management
-#  SEE https://pip.pypa.io/en/stable/reference/pip_install
-#
+{# SEE https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management
+#  SEE https://pip.pypa.io/en/stable/reference/pip_install #}
 FROM base as build
 
+ENV SC_BUILD_TARGET build
+
+{#TODO: install and uninstall lists#}
 RUN apk add --no-cache \
       postgresql-dev \
       libc-dev \
       libffi-dev \
       gcc \
       git
```

Comment:
do we really need all these package in the standard cookie? this is to be modified depending on needs.

Reply:
No, should we ask one by one in the cookiecutter query?? For the moment I left them all and left the user to decided what is actually needed

Reply:
I would tend to not bloat the service with things that are not necessary. It is easy to add whatever is needed later. I would either comment the unnecesary ones or delete them.

Reply:
Commented out